### PR TITLE
Removes the syndicate hand teleporter.

### PR DESCRIPTION
**Pull request by Pickle-Coding**
## About The Pull Request
Alternative to #74521

Removes the syndicate hand teleporter and its uplink entry.
## Why It's Good For The Game
The syndicate hand teleporter just simply can not work. There are other items traitors can use to break in to an area or navigate the station. The doorjack allows the traitor to break into any area locked behind an airlock, and allows them to exit back any time they want. The bluespace launchpad I don't really know how it works, but it allows the traitor to teleport into a room. A traitor could also steal a normal hand teleporter that can not be abused in control as it is completely random and not instant.

The syndicate hand teleporter offers what these things offer, but with the ability to be painfully abuseable. The syndicate hand teleporter can either be absurdly OP, or it could be clunky and only codedivers would know how to get around to actually use it. There is no need for this experimental hand teleporter that massively disrupts the ecosystem, as traitors have many methods of getting around places.
## Changelog
:cl:
balance: The syndicate hand teleporter is now missing from the uplink.
del: The syndicate hand teleporter no longer exists, and never existed.
/:cl:

### CODE REVIEW

**Comment by Addust** [comment]
get out you salty fuck

**Comment by Fikou** [comment]
@Addust discuss the pr on its terms, not interested in telling people to get out

**Comment by Pickle-Coding** [comment]
> get out you salty fuck

I am not salty and I haven't played much for a long time. I just don't think this concept of the syndicate hand teleporter can work.

**Comment by Helg2** [comment]
To be fair, the whole "why it's good for the game" part sounds like skill issue and you're just salty.

**Comment by Samag5** [comment]
Progressive traitor literally unplayable without syndi tele, you need to somehow run away from hordes of validhunting crew members while doing objectives in the hope that you will have time to make the final objective before the shuttle. If you want to delete syndi tele, then you also need to remove progressive traitor.

**Comment by SinfulBliss** [comment]
Very needed item for a system that for a long time was just brutal especially to new players. In the past tots just had to complete maybe 1 or 2 objectives then escape alive, it was pretty easy to be stealthy. Progtots have to complete this same amount of objectives just to unlock their default kit, and then they go on to complete several more over time.

More objectives = more chance of getting caught (which on LRP is usually getting looted and RR'd), syndie hand tele is a PURELY defensive item, it has no offensive potential whatsoever, and serves as a direct solution to the increased risk of getting caught progtot presented. So I completely disagree with the charge that it isn't needed. It's not used solely for breaking into places like the doorjack which is also significantly cheaper.

**Comment by Pickle-Coding** [comment]
> Progressive traitor literally unplayable without syndi tele, you need to somehow run away from hordes of validhunting crew members while doing objectives in the hope that you will have time to make the final objective before the shuttle. If you want to delete syndi tele, then you also need to remove progressive traitor.

Progression traitor existed before the syndicate teleporter. Many uplink items have had their reputation requirements removed since then. Do you consider relying on one uplink item to reliably get away from any situation is good gameplay?

I don't, because that's just reliably avoiding all engagement. If you couldn't teleport to the other side of the station at any given time, then maybe crew would be less bloodthirsty and angry at you, and more meaningful engagement could occur.

**Comment by Jacquerel** [comment]
> Very needed item for a system that for a long time was just brutal especially to new players. In the past tots just had to complete maybe 1 or 2 objectives then escape alive, it was pretty easy to be stealthy. Progtots have to complete this same amount of objectives just to unlock their default kit, and then they go on to complete several more over time.

Traitors on live also only need to complete two objectives and stay alive, any additional things they do are bonuses.

**Comment by SinfulBliss** [comment]
> > Very needed item for a system that for a long time was just brutal especially to new players. In the past tots just had to complete maybe 1 or 2 objectives then escape alive, it was pretty easy to be stealthy. Progtots have to complete this same amount of objectives just to unlock their default kit, and then they go on to complete several more over time.
> 
> Traitors on live also only need to complete two objectives and stay alive, any additional things they do are bonuses.

2-3 objectives for rep to unlock your kit + the two objectives to greentext are still twice as many objectives that tots have had to do, which is twice as many chances to get caught.

**Comment by 13spacemen** [comment]
Just make the syndie hand tele create portals so you can follow people through them, or nerf the charges and charge probability.
Traitors on TG have no teeth.
This exact item is on Bee and Paradise which are both MRP servers full of unrobust newbies, yet they haven't complained at all
I am willing to bet most of the people complaining about this item don't even bother playing Sec or try using EMP guns/nades

**Comment by moocowswag** [comment]
You why this pr is good for the game isnt really supportive of why this pr is good for the game, yes traitors have other methods to do a similar thing which means being able to do these sorts of things are fair?

the problem is the experimental teleporter is meant to be the cheap and unreliable, youre meant to make fewer but smarter uses of it to get the most out of it. Its only issue is its low cooldown / high uptime which is stopping it from becoming its own remotely balanced niche.

**Comment by Jacquerel** [comment]
> 2-3 objectives for rep to unlock your kit + the two objectives to greentext are still twice as many objectives that tots have had to do, which is twice as many chances to get caught.
> 
> (this is leaving aside the fact most traitors assume their job is to continuously complete objectives, so they will do them until they're caught or the shuttle comes)

Virtually everything which is neither a weapon nor an explosive does not require you to do that.
You don't need to do any objectives to unlock gear if you are willing to just wait instead. If people would rather have stuff sooner and give themselves away, they have made a choice and have to be prepared for that to have consequences.
If people want to choose to continuously complete objectives until they get caught that's their choice, and so facing the consequences of that choice (it is hard not to get caught when you are constantly doing nefarious acts) is natural.

The alternative to progression traitor would only be to remove the option to speed up the timer, the timer would not be removed.

**Comment by Pickle-Coding** [comment]
> Your why this pr is good for the game isnt really supportive of why this pr is good for the game, yes traitors have other methods to do a similar thing which means being able to do these sorts of things are fair?
> 
> 
> 
> the problem is the experimental teleporter is meant to be the cheap and unreliable, youre meant to make fewer but smarter uses of it to get the most out of it. Its only issue is its low cooldown / high uptime which is stopping it from becoming its own remotely balanced niche.
> 
> 
> 
> 
> 
> This is a balance issue that can most likely fixed with a change to several numbers, not a design flaw that has been aching the community for over a year

This isn't just about the balance, it's about the concept. I just don't think it's fun for either party if someone instantly teleports out of the situation as they please.

There is no RP, engagement, or anything. Nobody wins here, except for potential dopamine of completing objectives without interacting with anyone for the entire round.

Anything that solves this would just make it become another doorjack, briefcase launchpad, or hand teleporter. There is no point in wasting time balancing this when there is nothing to gain for anyone.

**Comment by SinfulBliss** [comment]
> Virtually everything which is neither a weapon nor an explosive does not require you to do that. You don't need to do any objectives to unlock gear if you are willing to just wait instead. 
Round times on LRP are range from 30 minutes to an hour, if you want to have any amount of time playing with your gear you need to complete at least a couple objectives to get the rep. The choice is less wanting it sooner and more wanting to be sure you'll get to play with it at all.

> If people want to choose to continuously complete objectives until they get caught that's also their choice, and so facing the consequences of that choice (it is hard not to get caught when you are constantly doing nefarious acts) is natural.
You're right, it's entirely a player issue, but the system feeds players into doing more objectives, so it's at least partly to blame for their decision. That's why more defensive tools like these are so valuable now IMO.

**Comment by Pickle-Coding** [comment]
For the record, everything is balanced around rounds being 90 minutes.

**Comment by 13spacemen** [comment]
> There is no RP, engagement, or anything. Nobody wins here, except for potential dopamine of completing objectives without interacting with anyone for the entire round.
If engagement is your issue then just make it create portals like the hand tele does so you can follow the traitor easily. Either that or reduce the charges/charge probability
Also you said earlier you had no idea how the bluespace launchpad works...it's an incredibly obvious item (briefcase, which nobody but the lawyer legitimately carries around) that takes ages to set up, can't be moved while it's set up, and usually results in you getting valided before you can finish setting it up

**Comment by Prussen** [comment]
> I don't, because that's just reliably avoiding all engagement. If you couldn't teleport to the other side of the station at any given time, then maybe crew would be less bloodthirsty and angry at you, and more meaningful engagement could occur.

People aren't going to validhunt you less because you lost the ability to reliably deal with stuns (and completing objectives will be significantly harder, who's gonna break into the CE's office with a doorjack (4 total doors to open) to kill Poly AND place a bug?). The crew will simply have an easier time killing you so they can get your tasty tot loot and the dopamine from RRing someone. Maybe there will be slightly less incentive now that theres no tele to grab, but theres still many tot items for them to snag.

**Comment by moocowswag** [comment]
> There is no RP, engagement, or anything. Nobody wins here, except for potential dopamine of completing objectives without interacting with anyone for the entire round.

this is extremely short sighted, your engagement is knowing where the person is and the direction they spent one of their precious teleports going, you can visually tell if they have eva protection or not, most stations are literally a circle when it comes to maneuvering around in it which inherently puts an advantage on the person chasing to at-least keep up with their target.

Believe it or not a fun and interactive cat and mouse game doesnt require you to be within 7 tiles of your target the entire time, its seeing where they arnt and estimating where they have gone in the time since you last saw them and if they left any clues behind between their limited jumps.

**Comment by DaydreamIQ** [comment]
Good change. Arguments for the teleporter boul down to "Skill issue lmao" . It really didn't need the absurd failsafe buff it got when it was ported from paradise. There's no skill behind it, nerf or remove and let the salt flow.

**Comment by 13spacemen** [comment]
> Good change. Arguments for the teleporter boul down to "Skill issue lmao" . It really didn't need the absurd failsafe buff it got when it was ported from paradise. There's no skill behind it, nerf or remove and let the salt flow.

It is exactly the same as it is on Paradise dude, 1:1 port. The reason the failsafe was too generous was due to a bug with teleport code's location finding proc

**Comment by moocowswag** [comment]
> There's no skill behind it, nerf or remove and let the salt flow.

Literally everyone ive seen has agreed it needs a nerf so i doubt people would be salty over that.

**Comment by Pickle-Coding** [comment]
> > There is no RP, engagement, or anything. Nobody wins here, except for potential dopamine of completing objectives without interacting with anyone for the entire round.
> 
> 
> 
> this is extremely short sighted, your engagement is knowing where the person is and the direction they spent one of their precious teleports going, you can visually tell if they have eva protection or not, most stations are literally a circle when it comes to maneuvering around in it which inherently puts an advantage on the person chasing to at-least keep up with their target.
> 
> 
> 
> Believe it or not a fun and interactive cat and mouse game doesnt require you to be within 7 tiles of your target the entire time, its seeing where they arnt and estimating where they have gone in the time since you last saw them and if they left any clues behind between their limited jumps. 
> 
> 
> 
> Tldr stop coding for NRP
> 
> 

If someone can teleport, then they can not get stuck, and they will have a mobility advantage. You can not engage with someone that has an unstoppable mobility advantage. Teleportation does not leave clues either. They were there, and now gone without a trace.

The item relies on its own special rng as its balance. It is very anti-sandbox, as the traitor has no need to use the station and the methods to get around a chase. It only engages with itself, not the rest of the game.

**Comment by Cheshify** [comment]
I've seen this tool abused and spammed and it's just not fun to interact with. I'd be supportive of a removal, nerf, or rework, because right now traitors can swoop across the station without a care for access, fighting anyone, or having any consequences.

**Comment by moocowswag** [comment]
> > > There is no RP, engagement, or anything. Nobody wins here, except for potential dopamine of completing objectives without interacting with anyone for the entire round.
> > 
> > 

> The item relies on its own special rng as its balance. It is very anti-sandbox, as the traitor has no need to use the station and the methods to get around a chase. It only engages with itself, not the rest of the game.

you mean like cultist teleport rod which is not only readily available to a conversion antag at the cost of some plasteel but is also extremely consistent and arguably stronger.

or like heretic blade shatter which really does put you somewhere random making you impossible to track especially good combined with void heretic since you can jaunt thru rooms or more easily replace the blades.

or like a circuit launchpad network which has its own outstanding downsides and outstanding upsides especially for something that is available to non antags.

the difference is all of these other things were marginally balanced in how much uptime they have, the cult rod has limited charges but you can print another free one every few minutes, the launchpad is easier to trace and has a large upfront cost, heretic blades replacability varies so certain paths can reload it better (but if youre using it as a weapon instead of baton and surgical saw youre doing it wrong)

all of these things do exactly what you say the experimental teleporter does except their uptime is limited or they have a massive cost. Thats what balance is gonna turn the experimental teleporter into and itl still fill its own niche even among the pack of everything i just listed.

**Comment by Helg2** [comment]
The whole argumentation is on the level of crabs bucket.

**Comment by Pickle-Coding** [comment]
> > > > There is no RP, engagement, or anything. Nobody wins here, except for potential dopamine of completing objectives without interacting with anyone for the entire round.
> 
> > > 
> 
> > > 
> 
> 
> 
> > The item relies on its own special rng as its balance. It is very anti-sandbox, as the traitor has no need to use the station and the methods to get around a chase. It only engages with itself, not the rest of the game.
> 
> 
> 
> you mean like cultist teleport rod which is not only readily available to a conversion antag at the cost of some plasteel but is also extremely consistent and arguably stronger.
> 
> 
> 
> or like heretic blade shatter which really does put you somewhere random making you impossible to track especially good combined with void heretic since you can jaunt thru rooms or more easily replace the blades.
> 
> 
> 
> or like a circuit launchpad network which has its own outstanding downsides and outstanding upsides especially for something that is available to non antags.
> 
> 
> 
> the difference is all of these other things were marginally balanced in how much uptime they have, the cult rod has limited charges but you can print another free one every few minutes, the launchpad is easier to trace and has a large upfront cost, heretic blades replacability varies so certain paths can reload it better (but if youre using it as a weapon instead of baton and surgical saw youre doing it wrong)
> 
> 
> 
> all of these things do exactly what you say the experimental teleporter does except their uptime is limited or they have a massive cost. Thats what balance is gonna turn the experimental teleporter into and itl still fill its own niche even among the pack of everything i just listed.
> 
> 
> 
> 

They all rely on an outside economy or structure. The whole point of the syndicate teleporter and whatever downside you can give it is so it could be abused if you use it right. If it couldn't, then it's just bloat at best, as there are other teleportation tools or methods to move around the station as a traitor, and the syndicate teleporter wouldn't add anything new to the roster.

**Comment by moocowswag** [comment]
> They all rely on an outside economy or structure. 

Like how traitors have a tc economy that is based on completing objectives?

Dont say you can buy it with round start tc because cultists can get their gear with round start plasteel or break down shutters for it (or tell a construct to break 3 walls)

>The whole point of the syndicate teleporter and 
whatever downside you can give it is so it could be abused if you use it right. 

Yes, abused as a tool that you sometimes/uncommonly use when you need it most (when balanced)

>If it couldn't, then it's just bloat at best, as there are other teleportation tools or methods to move around the station as a traitor, and the syndicate teleporter wouldn't add anything new to the roster.

it literally will fill its own niche as an item a traitor can buy for those reactive situations where they will need to quickly jump, basically an upgrade to just having a bs crystal and stabilized metal in your backpack (which non antags can get)

**Comment by moocowswag** [comment]
Anyway im done arguing, nerf the item into the ground itl still have a viable niche, not that you care.

**Comment by SomeoneYouProbablyKnow** [comment]
While on the topic of syndi teleporter tools. The Briefcase teleporter is a much cooler version of the concept. Maybe all that effort of protecting the hand tele should be funneled into making the items we DO have better and multi-functional. 

Briefcase launchpad deserves your love.

**Comment by Rex9001** [comment]
If you are removing this please replace it with something, this is a very interesting and unique item and I'd like to not just loose an option from the uplink

**Comment by moocowswag** [comment]
>Maybe all that effort of protecting the hand tele should be funneled into making the items we DO have better and multi-functional.

im back because this entirely new point/arguement reinvigorated me.

multi functional is how we get items like the emag

also the brief case launchpad is actually very versatile ranging from a utility item to a murder trap weapon

**Comment by Pickle-Coding** [comment]
> If you are removing this please replace it with something, this is a very interesting and unique item and I'd like to not just loose an option from the uplink

Briefcase launchpad.

**Comment by SomeoneYouProbablyKnow** [comment]
Emag is not multifunctional it's a bolated mess of all the ideas that didn't make it on their own. The "Misc" section.

The launchpad is already quite fun  yea(with the one glaring weakness of being a briefcase) 

Really what i'm sayin is that people are weirdly focused on the number of items in the uplink "if you remove it add something back". 

If they like some unique aspect of the hand tele that doesn't overlap with the launchpad or some other item i think they should try to expand the existing items to gain that functionality

**Comment by Helg2** [comment]
> Briefcase launchpad.

>im back because this entirely new point/arguement reinvigorated me.

**Comment by Pickle-Coding** [comment]
> > Briefcase launchpad.
> 
> 
> 
> >The bluespace launchpad I don't really know how it works

That is my problem, not yours.

**Comment by Rex9001** [comment]
> Emag is not multifunctional it's a bolated mess of all the ideas that didn't make it on their own. The "Misc" section.
> 
> The launchpad is already quite fun yea(with the one glaring weakness of being a briefcase)
> 
> Really what i'm sayin is that people are weirdly focused on the number of items in the uplink "if you remove it add something back".
> 
> If they like some unique aspect of the hand tele that doesn't overlap with the launchpad or some other item i think they should try to expand the existing items to gain that functionality

Overlapping items are fine, multiple items can serve the same function. You can have multiple sort of similar versions with this mindset dsword should be removed because it overlaps heavily with esword

**Comment by Helg2** [comment]
> That is my problem, not yours.

You're depending on it without knowing how it works.

**Comment by moocowswag** [comment]
> Really what i'm sayin is that people are weirdly focused on the number of items in the uplink "if you remove it add something back".

Because they dont fill the same niche we are losing an item that has its own niche for traitors.

The briefcase launchpad is less mobile and less useful for evasion, better for stealthy manipulation of the positions of mobs/items in a limited area where you set up.

The experimental teleporter could easily fill the niche as a very slow refilling gap opener when you desperately need to create space between you and your pursuers but with more accuracy than bs crystals because you spent tc for it.

**Comment by SomeoneYouProbablyKnow** [comment]
Not Desword and Esword they are different

But go ahead and tell me fists of the north star and esword are super duper different in fuction

**Comment by moocowswag** [comment]
> Not Desword and Esword they are different
> 
> But go ahead and tell me fists of the north star and esword are super duper different in fuction

They are different enough to where they are both kept, so the requirement to be a niche isnt that large.

but ill humor you, gloves of the north star is less loud (imagine if we had tts tho) and allows for synergizes like sleeping carp, can also not be disarmed (unless you are literally dis-armed) 

esword is literally your gatcha rng casino roll the dice to block a stun baton because you picked a loud weapon that everyone and their mother will recognize and HEAR THRU LAYERS OF WALLS.

**Comment by SomeoneYouProbablyKnow** [comment]
Honestly, fair. They are different. I think i came at this issue from a wrong angle, i'll have to rethink that. G'day to you and all that.

**Comment by CRITAWAKETS** [comment]
i don't get why the syndie hand teleporter is even a thing, a well-balanced version _already exists_ in the form of the briefcase teleport

**Comment by wawabenford** [comment]
instantly axing an item because its not balacned is a poor design philosiphy. how about you first try rebalancing in major ways such as a massive price hike, a removal of charge,s a removal of emergency teleports or even a long cooldown like the door jack.

**Comment by CMDR-Gungnir** [comment]
The problem with the syndie teleporter is that it completely removes ALL risk. You don't have to be careful, you don't have to worry that someone might be able to track you down. It's boring playing as a Det in a round when every tot's nabbed the teleporter because you're not going to be dumb enough to do it in front of someone, so without Witness Testimony all that's left is the scanner, which...oh hey, you just bypassed completely.

There should be some risk to doing objectives. The Teleporter nullifies it. If you think it's unplayable without it, this is a skill issue.

**Comment by zxaber** [comment]
I like the concept of the syndi teleporter, it's basically a jaunt button for some TC. I can't really give any reasons to keep it around except that I got one once (as a crew member, mind) and it was a lot of fun.

Straight up, I think it could stand to lose the safety mechanic entirely. You should be encouraged to obtain mesons to use it safely. Possibly also, it could lose the charge system and just have one cooldown between all teleports.

**Comment by GuillaumePrata** [comment]
The Teleporter makes engagements between security and traitors be extremelly boring and dragged out.
Instead of having a meaningful conflict security often needs to keep chasing and chasing and chasing the traitor.

Rewarding antagonists, the role made to bring conflict to a round, to play like boring cowards is just poor design.

**Comment by Rex9001** [comment]
I feel as if removal is redundant, nerf it instead its not really like the idea is unsalvegable. It just needs some nerfs

**Comment by IndieanaJones** [comment]
> The Teleporter makes engagements between security and traitors be extremely boring and dragged out. Instead of having a meaningful conflict security often needs to keep chasing and chasing and chasing the traitor.
> 
> Rewarding antagonists, the role made to bring conflict to a round, for playing like boring cowards is just poor design.

Security is at one of its peaks in strength compared to recent years and traitors are possibly at their lowest point ever. If traitors have to play like cowards in order to combat security's essentially one-shot melee, perhaps its better worth looking at security and their balance as opposed to traitors.

**Comment by Livrah** [comment]
> The Teleporter makes engagements between security and traitors be extremely boring and dragged out. Instead of having a meaningful conflict security often needs to keep chasing and chasing and chasing the traitor.
> 
> Rewarding antagonists, the role made to bring conflict to a round, for playing like boring cowards is just poor design.

Totally agree with lack of antag/sec conflict.

But core problem with boring antagonists is the design of the progressive traitor, not just one item.

**Comment by Striders13** [comment]
this item's probably the first non-nerf thing that happened to traitor since progtot. Traitors aren't fun to deal with cause they're weak as shit and you don't feel good about catching one (unless he's one of those 2k hours assistant players). Syndi tele's a crutch, but a needed one in the current climate.

**Comment by CMDR-Gungnir** [comment]
> this item's probably the first non-nerf thing that happened to traitor since progtot. Traitors aren't fun to deal with cause they're weak as shit and you don't feel good about catching one (unless he's one of those 2k hours assistant players). Syndi tele's a crutch, but a needed one in the current climate.

The problem is that it's a crutch that nullifies the rest of the game. How does this interact with the Forensics system? If you aren't dumb enough to get caught/seen by someone, you're not going to leave anything behind, which means the Detective's got no way to interact with you.

In fact, it's impossible for you to get caught using this, unless someone sees you do it. So you don't need to be careful and disguise your prints/fibers. You don't need to clean things after you're done.

**Comment by 13spacemen** [comment]
> The problem is that it's a crutch that nullifies the rest of the game. How does this interact with the Forensics system? If you aren't dumb enough to get caught/seen by someone, you're not going to leave anything behind, which means the Detective's got no way to interact with you.
> 
> In fact, it's impossible for you to get caught using this, unless someone sees you do it. So you don't need to be careful and disguise your prints/fibers. You don't need to clean things after you're done.

the solution to that is to make teleporting with it leave a "bluespace residue" decal with fibers/prints on it
not to remove the item entirely

**Comment by Prussen** [comment]
> The problem is that it's a crutch that nullifies the rest of the game. How does this interact with the Forensics system? If you aren't dumb enough to get caught/seen by someone, you're not going to leave anything behind, which means the Detective's got no way to interact with you.

And other forms of teleportation don't? The bluespace dust residue idea is great when it comes to making this game mechanic more engaging for others.

> In fact, it's impossible for you to get caught using this, unless someone sees you do it. So you don't need to be careful and disguise your prints/fibers. You don't need to clean things after you're done.

Clean up things after you're done? In most cases you don't have time to "clean up", it's not a skill issue, it's simply a time issue. You'll most likely get caught red handed unless it's 5 pop. Thermited through a reinforced wall? Broke a reinforced window? Doorjacked in? Good luck fixing that before someone sees you obviously covering up your trails. The only place where you might not get caught is the Captain's office, but then again, you can simply hack in through the one maints door protecting it.
And the experimental hand tele is not an instant "finish your objective card", if you're looking to place a bug or steal the station blueprints, you'll have to stay in the premises (you can't teleport the locker out with you), which still gives crew enough enough time to spot you through a window or at least walk by you as you very conspicuously teleport out.
I would support a nerf, like a longer cooldown, a reduction in charges and the bluespace residue idea. Removing the mechanic would make completing objectives significantly harder for traitors, which in the current landscape would nerf them quite a bit. (I don't think anybody would be having fun when the traitors are sitting in dorms lockers waiting for rep because they know they'll get caught doing objectives, lol)

**Comment by Helg2** [comment]
I mean, we can delete it so everyone will now understand that porgressive sucks ass.

**Comment by san7890** [comment]
I am not interested in this PR as I believe both #74521 and #74606 would appropriately address any grievances.
